### ISSUE TITLE
Disk usage differs from df command

### ISSUE THREAD

**Issue by ccztux** [NagiosEnterprises/ncpa#168]
Hi!

The disk usage in percentage differs from df -ah command.

check_npca.py output:
```
logical/
    |home/
        total_size: [2079906250752L, u'b']
        used_percent: [84.799999999999997, u'%']
        used: [1763922640896L, u'b']
        free: [294531969024L, u'b']
        device_name: [[u'/dev/mapper/VolGroup00-LogVol00'], u'name']
    |/
        total_size: [8318124032L, u'b']
        used_percent: [65.700000000000003, u'%']
        used: [5461069824L, u'b']
        free: [2427699200L, u'b']
        device_name: [[u'/dev/sda1'], u'name']
```

df -ah output:
```
Filesystem            Size  Used Avail Use% Mounted on
/dev/sda1             7.8G  5.1G  2.3G  70% /
proc                     0     0     0   -  /proc
sysfs                    0     0     0   -  /sys
devpts                   0     0     0   -  /dev/pts
/dev/mapper/VolGroup00-LogVol00
                      1.9T  1.7T  275G  86% /home
tmpfs                1006M     0 1006M   0% /dev/shm
none                     0     0     0   -  /proc/sys/fs/binfmt_misc
sunrpc                   0     0     0   -  /var/lib/nfs/rpc_pipefs
```

NCPA-Version: ncpa-1.8.0-1.el5.x86_64.rpm

Please let me know, if i could provide certain informations.

Best regards,

ccztux

**Comment by jhomann** [comment]
Not 100% sure why it is getting different amounts but the NCPA disk information is coming from python's module psutil and not from running df -ah or anything similar. I've also noticed a couple very small differences in what psutil returns vs what commands show.

**Comment by ccztux** [comment]
It is usefull to know, that these informations comes from psutil. A difference of 5% on a disk check is very bad.

**Comment by ccztux** [comment]
Maybe this is a duplicate: https://github.com/NagiosEnterprises/ncpa/issues/102

**Comment by krobertson71** [comment]
This is a dup of 102.  We still have this same issue for Linux RedHat servers.  I am hoping this gets addressed before 1.9.0.

**Comment by jhomann** [comment]
Closing this since it's a duplicate. Will be looking into this for next release too.
